_overviews/compiler-options/index.md

```diff
@@ -25,17 +25,13 @@ title: Scala Compiler Options
 
 ## Introduction
 
-Scala compiler `scalac` offers various **compiler options**, also referred to as **compiler flags**, to change how to compile your program.
+The Scala compiler `scalac` offers various **compiler options**, or **flags**, that change the compiler's default behavior. Some options just generate more compiler output in the form of diagnostics or warnings, while others change the result of compilation.
 
-Nowadays, most people are not running `scalac` from the command line.
-Instead, they use sbt, an IDE, and other tools as their interface to the compiler.
-Therefore they may not even have `scalac` installed, and won't think to do `man scalac`.
+The Scala command `scala`, which runs scripts or compiled code, accepts the compiler options plus a few more that determine how to run a program.
```

Comment:
perhaps change `the compiler options` to `all compiler options` ?

Reply:
or `most compiler options` if not exactly all are accepted?

Reply:
"the same options as the scalac compiler", where "the compiler options" meant "options to the compiler".